draft-ietf-quic-multipath.md

```diff
@@ -824,49 +828,44 @@ Path ID 1), and server provides two connection IDs
 as the Destination Connection ID of the new path.
 
 
 ## Path Closure
 
-In this example, the client detects a network environment change
-(client's 4G/Wi-Fi is turned off, Wi-Fi signal is fading below a threshold,
-or the quality of RTT or loss rate is becoming worse) and wants to close
-an existing path.
-
-{{fig-example-path-close1}} illustrates an example of path closing. For the first path, the
-server's 1-RTT packets use DCID C1 and the
-client's 1-RTT packets use DCID S1 for the path with Path ID 1. For the
-second path, the server's 1-RTT packets use DCID C2 and
-the client's 1-RTT packets use DCID S2 for the path with Path ID
-2.
-
-Note that the paths use different packet number spaces. In this case, the
-client is going to close the first path with Path ID 1 but it sends the
-ABANBON frame over the second path using the DCID S2. In this example, the
-server confirms the path closure by sending an PATH_ABANDON frame
-by for the same Path ID to the client also using the other path with DCID C2.
-Both the client and the server can close the path after receiving
-the MP_RETIRE_CONNECTION_ID frame for that path.
+{{fig-example-path-close1}} illustrates an example of path closure.
+
+In this example, the client wants to close the path with Path ID 1.
+It sends the PATH_ABANDON frame to terminate the path. After received
+the PATH_ABANDON frame containing Path ID 1, the server decide to close
+the path with sending PATH_ABANDON frame either. During this time,
+if endpoints still receive inflight packets except the acknowledgements,
+endpoints only sends PATH_ABANDON frame as reply or choose to ignore the inflight packets.
+After a short period of time (3 PTOs as specified in {#path-close}),
+both endpoints retire and free all the resources associated with
+the closed Path ID, and the closed Path ID MUST NOT be reused in the future.
 
 ~~~
 Client                                                      Server
 
-(client tells server to abandon a path)
-1-RTT[X]: DCID=S2 PATH_ABANDON[Path ID=1]->
+(client tells server to abandon a path with Path ID=1)
+1-RTT[X]: DCID=S1 PATH_ABANDON[Path ID=1]->
                            (server tells client to abandon a path)
-               <-1-RTT[Y]: DCID=C2 PATH_ABANDON[Path ID=1],
-                                               ACK_MP[PATH ID=1, PN=X]
-(client retires the corresponding CID)
-1-RTT[U]: DCID=S2 MP_RETIRE_CONNECTION_ID[Path ID=1, Seq=0],
-ACK_MP[Path ID=1, PN=Y] ->
-                            (server retires the corresponding CID)
-           <- 1-RTT[V]: DCID=C2 RETIRE_CONNECTION_ID[Path ID1, Seq=0],
-                                                ACK_MP[Path ID=1, PN=U]
+                    <-1-RTT[Y]: DCID=C1 PATH_ABANDON[Path ID=1],
+                                           ACK_MP[PATH ID=1, PN=X]
+1-RTT[U]: DCID=S1 ACK_MP[Path ID=1, PN=Y] ->
+(After 3 PTOs, client retires the corresponding CID)
+              (After 3 PTOs, server retires the corresponding CID)
```

Comment:
```suggestion
```
As this is not connected to any protocol actions, I don't think this is needed here anymore.